flow/CMakeLists.txt

```diff
@@ -46,6 +46,8 @@ set(FLOW_SRCS
   Knobs.cpp
   Knobs.h
   MetricSample.h
+  MkCert.h
```

Comment:
If we have dependencies on that code only in fdbrpc and above, I think it would be better to keep it there rather than in flow.

Reply:
The reason I decided to use **_flow_** and not **_fdbrpc_** to put **_MkCert_** is that RPC is a layer for FDB server-client protocols, and TLS is a more fundamental layer below that. Case in point, **_TLSConfig_** also currently resides in **_flow_**.